config/watcher_local_test/kustomization.yaml

```diff
@@ -37,6 +37,9 @@ patches:
     - op: add
       path: /spec/template/spec/containers/0/args/-
       value: --listener-port-overwrite=9443
+    - op: add
+      path: /spec/template/spec/containers/0/args/-
+      value: --skr-watcher-image=runtime-watcher-skr:latest
```

Comment:
this should be the same as in `skrwebhook/resources.yaml` right?

Reply:
or do we leave latest here and for the config/default we anyways overwrite with tag?

Reply:
As discussed let's keep testing latest here 👍